test/spec/api/layers.spec.js

```diff
@@ -222,6 +222,17 @@ describe('api.layers', function() {
 
       });
 
+      it("should load local viz.json correctly", function(done) {
+        sinon.spy($, "get");
+        cartodb.createLayer(map, "../../demos/viz.json")
+            .addTo(map);
+        setTimeout(function(){
+          expect($.get.getCall(0)); // If jquery get has been called, means it's loading it as a local, non jsonp vizjson
```

Comment:
I was using sinon, not jasmine in this case. I'm closing this PR and opening a new one with the more simple change :)